src/languages/en.ts

```diff
@@ -2061,6 +2061,11 @@ export default {
             lineItemLevel: 'Line-item level',
             reportLevel: 'Report level',
             appliedOnExport: 'Not imported into Expensify, applied on export',
+            shareNote: {
+                header: 'Easily share your workspace with other members.',
+                content:
```

Comment:
@dominictb I think it is better to divide this text into two parts:
1. `Share this QR code or copy the link below to make it easy for members to request access to your workspace. All requests to join the workspace will show up in the`
2. `room for your review.`

Reply:
With this approach, we don't need to apply logic to split the content in the component